### PR TITLE
test: add editable heading interaction test

### PR DESCRIPTION
**Pull request by jes14**
Update <EditableHeading> interaction tests (https://github.com/mondaycom/monday-ui-react-core/issues/1576)

### CODE REVIEW

File: src/components/EditableHeading/__tests__/editable-heading-interactions.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  fireEvent.change(editableHeadingElement, { target: { value: "Typed Text" } });
+  expect(editableHeadingElement).toHaveAttribute("value", "Typed Text");
+}
+
+export const statesPlaySuite = interactionSuite({
+  tests: [states_editableHeadingElementTest],
+  afterEach: async () => {
+    await resetFocus();
+  }
+});
```
Comment (SergeyRoyt) [inline src/components/EditableHeading/__tests__/editable-heading-interactions.tsx:19]: Tried it out, but it didn't worked correctly
Used it as a draft though. Propose to change it like this

```suggestion
import { expect } from "@storybook/jest";
import {
  Canvas,
  clearText,
  clickElement,
  getByTestId,
  interactionSuite,
  typeText
} from "../../../tests/interactions-utils";
import { ComponentDefaultTestId } from "../../../tests/constants";
import { resetFocus } from "../../../__tests__/interactions-helper";

const getEditableHeadingHeading = async (canvas: Canvas) => {
  await resetFocus();
  return getByTestId(canvas, ComponentDefaultTestId.HEADING);
};

const getEditableHeadingInput = async (canvas: Canvas) => {
  await resetFocus();
  const editableHeadingElement = getByTestId(canvas, ComponentDefaultTestId.CLICKABLE);
  clickElement(editableHeadingElement);
  return getByTestId(canvas, ComponentDefaultTestId.EDITABLE_INPUT);
};

async function textSimpleText(canvas: Canvas) {
  const editableHeadingInput = await getEditableHeadingInput(canvas);
  await clearText(editableHeadingInput);

  const text = "Typed Text";
  await typeText(editableHeadingInput, text);
  expect(editableHeadingInput).toHaveAttribute("value", text);

  await resetFocus();
  const editableHeading = await getEditableHeadingHeading(canvas);
  expect(editableHeading).toHaveTextContent(text);
}

export const overviewPlaySuite = interactionSuite({
  tests: [textSimpleText]
});
```

File: src/components/EditableHeading/__stories__/EditableHeading.stories.mdx [hunk 2]
```diff
@@ -72,7 +72,7 @@ An extension of Heading component, it allows built in editing capabilities
 
 <Canvas>
   <Story name="Placeholder">
-    <EditableHeading type={EditableHeading.types.h1} placeholder="H1 Placeholder" />
+    <EditableHeading type={EditableHeading.types.h1} placeholder="H1 Placeholder" ariaLabel="editableHeading"/>
```
Comment (SergeyRoyt) [inline src/components/EditableHeading/__stories__/EditableHeading.stories.mdx:75]: This one is not really needed

File: src/components/EditableHeading/__stories__/EditableHeading.stories.mdx
Comment (SergeyRoyt) [inline src/components/EditableHeading/__stories__/EditableHeading.stories.mdx:1]: Here to `<Story name="Overview"` you can add `play={overviewPlaySuite}`, this way the storybook will run this interaction test and you will be able to see it on the `Sandbox` tab. It'll also run automatically while running tests in Chromatic

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
Didn't work for me, proposed the fixed

**Comment by jes14** [comment]
Hi, @SergeyRoyt 
Thank you for your feedback. Your review was really helpful. I've updated from the feedback -> ready for review

**Review by SergeyRoyt — APPROVED** [review]
Great!
Thanks for the contribution, @jes14!

**Pull request merged**